R/params.R

```diff
@@ -175,6 +175,13 @@ params_configurable <- function(param) {
   length(param$value) <= 1 && !is.null(params_get_control(param))
 }
 
+# Returns a new empty named list.
+params_namedList <- function() {
+  empty <- list()
+  names(empty) <- character()
+  empty
+}
```

Comment:
Wow. I am awesome. My brain didn't even pick up on that. I was, "oh. we prefixed all the params stuff with `params_` so i'll do that" and then, "oh. RJSONIO has `emptyNamedList` so i'll follow their lead".